pkg/system/phase4_configuring.go

```diff
@@ -1281,12 +1341,23 @@ func (r *Reconciler) ReconcileDefaultBucketClass() error {
 		return nil
 	}
 
-	r.DefaultBucketClass.Spec.PlacementPolicy = &nbv1.PlacementPolicy{
-		Tiers: []nbv1.Tier{{
-			BackingStores: []nbv1.BackingStoreName{
-				r.DefaultBackingStore.Name,
+	if !util.IsFusionHCIWithScale() {
+		r.DefaultBucketClass.Spec.PlacementPolicy = &nbv1.PlacementPolicy{
+			Tiers: []nbv1.Tier{{
+				BackingStores: []nbv1.BackingStoreName{
+					r.DefaultBackingStore.Name,
+				},
+			}},
+		}
+	} else {
+		r.Logger.Info("IBM Fusion HCI with spectrum scale detected.")
+		r.DefaultBucketClass.Name = "noobaa-default-nsfs-bucket-class"
```

Comment:
I prefer keeping the same name, as we will probably have other default namespace stores.

Reply:
done.